src/test/java/bio/terra/service/snapshot/SnapshotIntegrationTest.java

```diff
@@ -92,23 +92,23 @@ public void setup() throws Exception {
 
   @After
   public void tearDown() throws Exception {
-    createdSnapshotIds.forEach(
-        snapshot -> {
-          try {
-            dataRepoFixtures.deleteSnapshot(steward(), snapshot);
-          } catch (Exception ex) {
-            logger.warn("cleanup failed when deleting snapshot " + snapshot);
-            ex.printStackTrace();
-          }
-        });
-
-    if (datasetId != null) {
-      dataRepoFixtures.deleteDatasetLog(steward(), datasetId);
-    }
-
-    if (profileId != null) {
-      dataRepoFixtures.deleteProfileLog(steward(), profileId);
-    }
+    //    createdSnapshotIds.forEach(
+    //        snapshot -> {
+    //          try {
+    //            dataRepoFixtures.deleteSnapshot(steward(), snapshot);
+    //          } catch (Exception ex) {
+    //            logger.warn("cleanup failed when deleting snapshot " + snapshot);
+    //            ex.printStackTrace();
+    //          }
+    //        });
+    //
+    //    if (datasetId != null) {
+    //      dataRepoFixtures.deleteDatasetLog(steward(), datasetId);
+    //    }
+    //
+    //    if (profileId != null) {
+    //      dataRepoFixtures.deleteProfileLog(steward(), profileId);
+    //    }
```

Comment:
Did you mean to leave this commented out?

Reply:
Ah this was from testing! Thanks for catching this!